apps/src/sites/studio/pages/blocks/index.js

```diff
@@ -50,7 +50,7 @@ $(document).ready(() => {
   Blockly.assetUrl = assetUrl;
   Blockly.valueTypeTabShapeMap = valueTypeTabShapeMap(Blockly);
   Blockly.typeHints = true;
-  Blockly.Css.inject(document);
+  Blockly.Css.inject(true, 'media');
```

Comment:
@molly-moen - I am working on a PR for rendering blocks in correct blockly version on the levelbuilder edit page. Could you please explain line 53 above and why you can use this call for both 'Google' and 'Cdo' versions? Thank you.

Reply:
https://codedotorg.slack.com/archives/C03DBDN67B7/p1673457806422289?thread_ts=1671051857.952909&cid=C03DBDN67B7